fdbserver/workloads/ConfigureDatabase.actor.cpp

```diff
@@ -349,7 +351,11 @@ struct ConfigureDatabaseWorkload : TestWorkload {
 			}
 			state int randomChoice;
 			if (self->allowTestStorageMigration) {
-				randomChoice = deterministicRandom()->randomInt(4, 9);
+				randomChoice = (deterministicRandom()->random01() < 0.375) ? deterministicRandom()->randomInt(0, 3)
```

Comment:
Why is 3 an invalid value here?

Reply:
Because 3 will change the replica setting. It's possible at first the cluster has 3 storages and `single` replica, where perpetual wiggle can make progress, then a `configure triple` is issued, which makes perpetual wiggle paused because the storages are not enough for perpetual wiggle to happen. We need at least 4 storages in `triple` scenario.